### PR TITLE
Login modal

### PR DESCRIPTION
**Pull request by Xesme**
This incorporates fixes for issue #176 and in reference to issues #170 and #175.

### CODE REVIEW

File: app/src/scripts/nav/navbar.jsx [hunk 1]
```diff
@@ -11,9 +11,10 @@ import actions from '../user/user.actions.js'
 import { Navbar } from 'react-bootstrap'
 import { Modal } from '../utils/modal.jsx'
 import { refluxConnect } from '../utils/reflux'
-
 import brand_mark from './assets/brand_mark.png'
 
+import logo_app from './assets/logo_app.png'
```
Comment (nellh) [inline app/src/scripts/nav/navbar.jsx]: Looks like this an the related new asset are unused and can be removed.

**Review by nellh — CHANGES_REQUESTED** [review]
The modal looks good and I tested login/logout with both providers.

We should revert the styling for the button in the navbar now that it is a button again (blue call to action).

When you logout, the modal remains open. This state should get reset so the modal doesn't block the page after logout.

File: app/src/scripts/nav/navbar.jsx (hunk 1 above)
Comment (Xesme) [reply app/src/scripts/nav/navbar.jsx]: Updates have been pushed to fix these issues.

**Review by nellh — APPROVED** [review]

**Comment by nellh** [comment]
I think we should merge this after the 1.2 release tomorrow and include it with the other login improvements in the next release, but this is good. Thanks @Xesme

**Comment by Xesme** [comment]
@nellh sounds great. I would like to add the helper-text to this branch as well, we can talk about it when you get in today. 👍

**Pull request merged**